package.json

```diff
@@ -33,10 +36,16 @@
   },
   "homepage": "https://github.com/gagern/node-libtidy#readme",
   "devDependencies": {
+    "@types/chai": "^3.5.0",
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
In that case better use the mismatched version. As far as I can tell, the TypeScript dependencies don't pull in the actual libraries, so we shouldn't see much impact from the mismatch unless we start using API which is different between those versions. Right?

A Pull Request against the DefinitelyTyped might be a good idea if you want to, but I think we don't block on that here.